documentation/coding_guidelines.md

```diff
@@ -181,7 +181,7 @@ Error handling suggestions follow the [Rust book guidance](https://doc.rust-lang
 
 *Panic*
```

Comment:
can you put in a blurb about infallible here?

Reply:
yes sir!

Reply:
how about that?